### ISSUE TITLE
Add e2e test for simple keyboard navigation through blocks

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#12392]
After a serious breakage to keyboard navigation through the blocks reported in https://github.com/WordPress/gutenberg/issues/11773 and solved https://github.com/WordPress/gutenberg/pull/12390 during Release Candidate time, [there is consensus](https://wordpress.slack.com/archives/C02QB2JS7/p1543430235127000) an e2e test specific to check keyboard navigation through blocks would be valuable. 

The bug report on #11773 followed the following workflow, maybe it could be used as a basis for the test:

- create 3 paragraphs blocks with some content
- navigating with the Tab key should navigate the 3 blocks UI as expected, in the expected order which at the moment is:
  - block external focusable wrapper
  - add block button
  - block movers
  - block toolbar
  - block editable area
- note: this order might change in the future so it would be great if the test could be easily adjusted 
- move to another part of the Gutenberg UI, for example using the navigate regions shortcut or clicking elsewhere as reported in #11773 
- repeat keyboard navigation through the 3 blocks to verify everything works as expected

**Comment by afercia** [comment]
Fixed in #13455
